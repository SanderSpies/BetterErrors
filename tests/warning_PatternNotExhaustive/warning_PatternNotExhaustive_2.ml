type greetings =
  | Hello
  | Goodbye
  | Hola of string

let say a = match a with
| Hello -> ()
