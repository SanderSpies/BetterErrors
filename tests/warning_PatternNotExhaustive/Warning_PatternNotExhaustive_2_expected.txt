[36mtests/Warning_PatternNotExhaustive/Warning_PatternNotExhaustive_2.ml:6 12-17
[0m4 |   | Hola of string
5 | 
6 | let say a = [31mmatch[0m a with
                ^^^^^
7 | | Hello -> ()

Warning 8: this match doesn't cover all possible values of the variant.
These cases are not matched:
- `Goodbye`
- `Hola _`
